### PR TITLE
escalation: tuberose

### PR DESCRIPTION
**Pull request by suanjiansalt**
by me. please give it a check at least.

### CODE REVIEW

File: contractdata/BANGKOK/ESCALATION/TUBEROSE/TUBEROSE1.json [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+                                }
+                            ]
+                        }
+                    }
+                }
+            }
+        ],
+        "Bricks": [],
+        "GameChangers": [
+            "9a89c97d-532b-e8b2-6e34-312470321f2e"
```
Comment (AnthonyFuller) [inline contractdata/BANGKOK/ESCALATION/TUBEROSE/TUBEROSE1.json:267]: This Gamechanger does not exist.

**Review by AnthonyFuller — CHANGES_REQUESTED** [review]

**Comment by suanjiansalt** [comment]
And if I get it correctly, the weird blank objective may be caused by gamechangers it uses.

File: contractdata/BANGKOK/ESCALATION/TUBEROSE/TUBEROSE1.json (hunk 1 above)
Comment (suanjiansalt) [reply contractdata/BANGKOK/ESCALATION/TUBEROSE/TUBEROSE1.json:267]: Updated adapted gamechangers.

**Comment by AnthonyFuller** [comment]
On Level 1, when pouring the insecticide, it instantly fails. I was wearing the Hotel Staff disguise.

**Comment by AnthonyFuller** [comment]
Level 2:
image

image

**Comment by AnthonyFuller** [comment]
Also please ensure you are following this [escalation guidelines](https://github.com/thepeacockproject/Peacock/blob/v7/docs/ESCALATION_GUIDELINES.md). I will check this myself, but it would save me some time.

**Comment by suanjiansalt** [comment]
> Also please ensure you are following the [escalation guidelines](https://github.com/thepeacockproject/Peacock/blob/v7/docs/ESCALATION_GUIDELINES.md).

I checked the guideline (or shall I say that's the rule I follow when making the contract) and the only disobey is in the last contract to kill the woman, it automatically appears in the last position due to activition machanics.

**Comment by AnthonyFuller** [comment]
> Also many descriptions that don't use full stops, this is to minimize the file size by reusing strings.
That's fine.

Also level 2:
image

**Comment by riisikumi** [comment]
I can't say anything about the escalation itself, but I'd suggest changing the name – The Trading Tradition sounds a bit dumb imo

**Comment by suanjiansalt** [comment]
It's called The Ethereal Eruption now. (Maybe dumber?)

**Review by AnthonyFuller — APPROVED** [review]
LGTM. Please PR images into the image pack.

**Comment by AnthonyFuller** [comment]
Upon further discussion internally, this escalation won't be merged into core. We recommend making it a mod + plugin and release it on Nexus.

**Comment by RDIL** [comment]
Will just put this here for context as to why I'm not wanting to add this one to core, I think it's an okay escalation, it's well done for your first time making one! There are a few reasons I don't want to add this one to core, namely, we used to have a modded bonus mission in Peacock called Traditions of the Trade by kercyx which played similarly to this escalation, and people did not like it.

I'm not saying this escalation is bad, it's more that I'm not sure if adding an escalation which had a very similar idea to content we already removed is a good idea, just because it might feel too similar.

Appreciate the PR nonetheless! The Hawkes Bay escalation in #402 looks really good and I'm excited to get that one merged!

**Comment by suanjiansalt** [comment]
> We used to have a modded bonus mission in Peacock called Traditions of the Trade by kercyx which played similarly to this escalation, and people did not like it.
> 
> I'm not saying this escalation is bad, it's more that I'm not sure if adding an escalation which had a very similar idea to content we already removed is a good idea, just because it might feel too similar.
> 

Well in fact that's from which I had an idea to make this escalation. A well-designed level (of course it has a very bad experience, too long and tedious) is quite simply discarded. Of course it's not even my brainchild so I'm just a little bit sorry about that. Certainly I wouldn't say, "Oh, my adaptation would be a thousand times stronger!" but just want something to live even in another forms.

And - of course, I'd say the other "bonus mission" (Lee Hong one) doesn't deserve much sympathy from me because it's so similar to the official escalation.

**Comment by AnthonyFuller** [comment]
Yeah Lee Hong was removed for that reason too. I did play the escalation, I do recommend making it a mod. You can't add localised lines via a plugin.

**Pull request closed without merging**